core/src/test/scala/org/apache/pekko/persistence/jdbc/state/scaladsl/DurableStateStorePluginSpec.scala

```diff
@@ -11,14 +11,23 @@ package org.apache.pekko.persistence.jdbc.state.scaladsl
 
 import com.typesafe.config.{ Config, ConfigFactory }
 import org.apache.pekko
+import org.apache.pekko.persistence.jdbc.config.SlickConfiguration
```

Comment:
* with these imports, remove the org.apache bit
* we have already have `import org.apache.pekko`
* please order them below the `import pekko.actor._`